cmd/admin_auth_ldap.go

```diff
@@ -273,6 +301,27 @@ func parseLdapConfig(c *cli.Context, config *ldap.Source) error {
 	if c.IsSet("skip-local-2fa") {
 		config.SkipLocalTwoFA = c.Bool("skip-local-2fa")
 	}
+	if c.IsSet("enable-groups") {
+		config.GroupsEnabled = c.Bool("enable-groups")
+	}
+	if c.IsSet("group-search-base") {
+		config.GroupDN = c.String("group-search-base")
+	}
+	if c.IsSet("group-member-uid") {
+		config.GroupMemberUID = c.String("group-member-uid")
+	}
+	if c.IsSet("group-user-attribute") {
+		config.UserUID = c.String("group-user-attribute")
+	}
+	if c.IsSet("group-filter") {
+		config.GroupFilter = c.String("group-filter")
+	}
+	if c.IsSet("group-team-map") {
+		config.GroupTeamMap = c.String("group-team-map")
+	}
+	if c.IsSet("group-users-remove") {
+		config.GroupTeamMapRemoval = c.Bool("group-users-remove")
+	}
```

Comment:
Sorry for the new review  .... just checked the UI, do we need to use the same phrase as UI?

* `Group Search Base DN` => `--group-search-base: The LDAP base DN at which group accounts will be searched for`?
* `group-member-uid` vs `group-user-attribute`: should we always use `uid` for consistency? Or always use `attribute` to match UI?
* `group-users-remove`: it belongs to `group-team-map`, so maybe it could be `group-team-map-removal`?